tests/src/test/java/alluxio/client/rest/AlluxioMasterRestApiTest.java

```diff
@@ -102,93 +113,316 @@ public void getConfiguration() throws Exception {
 
   private void checkConfiguration(PropertyKey key, String expectedValue, Map<String, String> params)
       throws Exception {
-    assertEquals(expectedValue, getInfo(params).getConfiguration().get(key.toString()));
+    Assert.assertEquals(expectedValue, getInfo(params).getConfiguration().get(key.toString()));
   }
 
   @Test
   public void getLostWorkers() throws Exception {
     List<WorkerInfo> lostWorkersInfo = getInfo(NO_PARAMS).getLostWorkers();
-    assertEquals(0, lostWorkersInfo.size());
+    Assert.assertEquals(0, lostWorkersInfo.size());
   }
 
   @Test
   public void getMetrics() throws Exception {
-    assertEquals(Long.valueOf(0),
+    Assert.assertEquals(Long.valueOf(0),
         getInfo(NO_PARAMS).getMetrics().get(MetricsSystem.getMetricName("CompleteFileOps")));
   }
 
   @Test
   public void getMountPoints() throws Exception {
     Map<String, MountPointInfo> mountTable = mFileSystemMaster.getMountTable();
     Map<String, MountPointInfo> mountPoints = getInfo(NO_PARAMS).getMountPoints();
-    assertEquals(mountTable.size(), mountPoints.size());
+    Assert.assertEquals(mountTable.size(), mountPoints.size());
     for (Map.Entry<String, MountPointInfo> mountPoint : mountTable.entrySet()) {
-      assertTrue(mountPoints.containsKey(mountPoint.getKey()));
+      Assert.assertTrue(mountPoints.containsKey(mountPoint.getKey()));
       String expectedUri = mountPoints.get(mountPoint.getKey()).getUfsUri();
       String returnedUri = mountPoint.getValue().getUfsUri();
-      assertEquals(expectedUri, returnedUri);
+      Assert.assertEquals(expectedUri, returnedUri);
     }
   }
 
   @Test
   public void getRpcAddress() throws Exception {
-    assertTrue(getInfo(NO_PARAMS).getRpcAddress()
-        .contains(String.valueOf(NetworkAddressUtils.getPort(ServiceType.MASTER_RPC,
-            ServerConfiguration.global()))));
+    Assert.assertTrue(getInfo(NO_PARAMS).getRpcAddress().contains(String.valueOf(
+        NetworkAddressUtils.getPort(ServiceType.MASTER_RPC, ServerConfiguration.global()))));
   }
 
   @Test
   public void getStartTimeMs() throws Exception {
-    assertTrue(getInfo(NO_PARAMS).getStartTimeMs() > 0);
+    Assert.assertTrue(getInfo(NO_PARAMS).getStartTimeMs() > 0);
```

Comment:
we prefer `assertX` over `Assert.assertX`

Reply:
done